src/main/java/com/stripe/model/EventDataDeserializer.java

```diff
@@ -85,13 +85,24 @@ public EventData deserialize(JsonElement json, Type typeOfT, JsonDeserialization
           eventData.setPreviousAttributes(previousAttributes);
         }
       } else if ("object".equals(key)) {
-        String type = element.getAsJsonObject().get("object").getAsString();
-        Class<? extends StripeObject> cl = EventDataClassLookup.findClass(type);
-        StripeObject object = ApiResource.GSON.fromJson(
-            entry.getValue(), cl != null ? cl : StripeRawJsonObject.class);
-        eventData.setObject(object);
+        eventData.setObject(entry.getValue().getAsJsonObject());
```

Comment:
`EventData` no longer tries to deserialize risky class-specific JSON, and it is exposed in `EventVersionedData` instead making using of the method below `deserializeStripeObject`